### PR TITLE
Add some extra clarifications on docs concerning linked docker services

### PR DESCRIPTION
**Pull request by bartvanb**

### CODE REVIEW

File: docs/features/node/linked_docker_service.rst [hunk 1]
```diff
@@ -20,7 +20,14 @@ where `container_name` is the name of your Docker container. This container will
 be made available in the Docker network where the algorithm containers are
 running, so your algorithm will be able to access it via `http://localhost`.
 The `container_label` will be used as alias for the container in the isolated
-Docker network.
+Docker network. You can then reach the container from within the algorithm container at
+the following address:
+
+.. code:: python
```
Comment (frankcorneliusmartin) [inline docs/features/node/linked_docker_service.rst:26]: This code example is specifix for http. Maybe good to mention that this also works for other protocols.

File: docs/features/node/linked_docker_service.rst [hunk 2]
```diff
@@ -20,7 +20,14 @@ where `container_name` is the name of your Docker container. This container will
 be made available in the Docker network where the algorithm containers are
 running, so your algorithm will be able to access it via `http://localhost`.
```
Comment (frankcorneliusmartin) [inline docs/features/node/linked_docker_service.rst:21]: Then this is also not a true statement, right?

**Review by frankcorneliusmartin — CHANGES_REQUESTED** [review]

File: docs/features/node/linked_docker_service.rst (hunk 2 above)
Comment (bartvanb) [reply docs/features/node/linked_docker_service.rst:21]: True, removed it

File: docs/features/node/linked_docker_service.rst (hunk 1 above)
Comment (bartvanb) [reply docs/features/node/linked_docker_service.rst:26]: done

**Comment by CunliangGeng** [comment]
Did you update Ubuntu version for https://docs.vantage6.ai/en/main/node/requirements.html?

**Comment by bartvanb** [comment]
> Did you update Ubuntu version for https://docs.vantage6.ai/en/main/node/requirements.html?

Yes, see [0b4cba1](https://github.com/vantage6/vantage6/pull/1329/commits/0b4cba10c7231e6e534cfc0f0d965dfc3294baf0) :-)

**Pull request merged**